indico/MaKaC/PDFinterface/base.py

```diff
@@ -708,8 +709,21 @@ def __init__(self):
         latex_mdx = mdx_latex.LaTeXExtension()
         latex_mdx.extendMarkdown(md, markdown.__dict__)
 
+        # Escape LaTeX before markdown
+        def _convert_markdown(text):
+            math_segments = []
+            placeholder = u"\uE000"
+
+            def _math_replace(m):
+                math_segments.append(m.group(0))
+                return placeholder
+
+            text = re.sub(r'\$[^\$]+\$|\$\$(^\$)\$\$', _math_replace, text)
```

Comment:
`latex_escape` in mdx_latex.py seems to do almost the same thing - why do we have to do it here manually? Bug in mdx_latex?